tests/cypress/integration/ctb.cy.js

```diff
@@ -0,0 +1,151 @@
+// <reference types="Cypress" />
+const products = require('../fixtures/products.json');
+const ctbGET = require('../fixtures/ctbGET.json');
+const ctbPOST = require('../fixtures/ctbPOST.json');
+
+describe('Click to buy', function () {
+
+	before(() => {
+		cy.intercept({
+			method: 'GET',
+			url: /newfold-marketplace(\/|%2F)v1(\/|%2F)marketplace/
+		}, products ).as('products');
+
+		cy.visit('/wp-admin/admin.php?page=' + Cypress.env('pluginid') + '#/marketplace', {
```

Comment:
@circlecube We should determine the desired convention for naming environment variables. Since we're already using camelcase, I propose renaming `pluginid` to `pluginId`.

Reply:
Good callout, thanks!